src/effects/mod.rs

```diff
@@ -10,6 +10,16 @@ pub mod prelude {
     pub use super::all_effects_request::*;
     pub use super::effects_response::*;
     pub use super::effects_for_account_request::*;
-    pub use super::effects_for_account_response::*;
+    pub use super::effects_for_liquidity_pools_request::*;
     pub use super::effects_for_ledger_request::*;
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn dummy_test() {
+        assert_eq!(EFFECTS_PATH, "effects");
+    }
+}
```

Comment:
Misschien moeten we deze er nog even uit gooien (die had ik alleen toegevoegd om de code coverage omhoog te krijgen :see_no_evil: )